Source/JavaScriptCore/llint/InPlaceInterpreter.asm

```diff
@@ -4302,7 +4293,7 @@ unimplementedInstruction(_simd_i64x2_extmul_high_i32x4_u)
 
 unimplementedInstruction(_simd_f32x4_abs)
 unimplementedInstruction(_simd_f32x4_neg)
-reservedOpcode(0xFDC201)
+reservedOpcode(0xFDE201)
```

Comment:
This change seems mysterious. Change log doesn’t mention it. Is the reason for it obvious to people who have more expertise than I do?

Reply:
I think it's a copy paste error or typo. I don't think it caused problems before because prior to this change the passed value to `reservedOpcode` was ignored. Now `reservedOpcode` adds a global label so there would be a conflict with the previous `reservedOpcode(0xFDC201)` a few lines above.

Reply:
@kmiller68 is correct, that's the reason. The new opcode should be the correct one to go in place.